utils/wait.sh

```
@@ -12,7 +12,7 @@ function wait_for_start {
     do
       if [[ ${COUNT} -gt 50 ]]; then
           echo "Failed to start Stardog cluster on time"
-          return 1;
+          exit 1;
```

fail hard if we can't start a cluster.